schema_utils.py

```diff
@@ -627,3 +627,23 @@ def is_valid_audio_language_code(obj):
             bool. Whether the given object is a valid audio language code.
         """
         return utils.is_valid_language_code(obj)
+
+    @staticmethod
+    def is_valid_thread_id(obj):
+        # type: (Text) -> bool
+        """Checks if the given obj (a string) represents a valid thread id.
+        The id has the form [entity_type].[entity_id].[generated_string]
+
+        Args:
+            obj: str. The thread id to verify.
+
+        Returns:
+            bool. Whether the given object is a valid thread id.
+        """
+        thread_id_must_starts_with = [
+            'exploration.', 'collection.', 'story.', 'learntopic.']
```

Comment:
aren't these already defined somewhere?

Reply:
https://github.com/oppia/oppia/pull/13355#discussion_r672776206 please take a look in this comment.